fpga_builder/builder.py

```diff
@@ -476,10 +513,17 @@ def get_stats_file(run_dir, num_threads):
         A unique path to a text file that can be populated with stats about the build
 
     """
+    import platform
+    import os
+    platform_sys = platform.system()
+    if (platform_sys == "Linux"):
```

Comment:
Looking online, this is because pathlib .resolve() changes it to a UNC path.  We can wrap this into a special thing that fixes the problem.  But because it's a wrapper anyways I don't find it super important what the contents of that wrapper are, as long as the code isn't everywhere.  Can we put this code into a function in utils.py and call that function instead of resolve where it is needed?